com.tradingview.tw.yaml

```diff
@@ -0,0 +1,53 @@
+id: com.tradingview.tw
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+separate-locales: false
+command: tradingview
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  # Files
+  - --filesystem=xdg-download:ro
+
+modules:
+  - name: tradingview
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply_extra -t $FLATPAK_DEST/bin/
+      - install -Dm755 tradingview.sh $FLATPAK_DEST/bin/tradingview
+      - install -Dm644 com.tradingview.tw.metainfo.xml -t $FLATPAK_DEST/share/metainfo
+      - install -Dm644 com.tradingview.tw.png -t $FLATPAK_DEST/share/icons/hicolor/256x256/apps
+      - install -Dm644 com.tradingview.tw.desktop -t $FLATPAK_DEST/share/applications
+    sources:
+      - type: script
+        commands:
+          - bsdtar -O -xf tradingview_amd64.deb data.* | bsdtar --no-same-owner -xf -
+          - mv opt/TradingView tradingview
+          - rm -rf tradingview_amd64.deb etc usr opt
+        dest-filename: apply_extra
+
+      - type: file
+        path: tradingview.sh
+
+      - type: file
+        path: com.tradingview.tw.metainfo.xml
+
+      - type: file
+        path: com.tradingview.tw.png
+
+      - type: file
+        path: com.tradingview.tw.desktop
+
+      - type: extra-data
+        filename: tradingview_amd64.deb
+        only-arches: [x86_64]
+        url: https://tvd-packages.tradingview.com/ubuntu/stable/latest/jammy/tradingview_amd64.deb
```

Comment:
This is not versioned. So it will break all the time.

Reply:
TradingView only provided this download link, can I manually handle it if there are any updates in the future?

Reply:
323a8f782aaf6ac58e89b2857d2e66190a6595d5 updated